noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -277,7 +278,7 @@ fn note_hash_exists(note_hash: Field, leaf_index: Field) -> u8 {}
 fn emit_note_hash(note_hash: Field) {}
 
 #[oracle(avmOpcodeNullifierExists)]
-fn nullifier_exists(nullifier: Field) -> u8 {}
+fn nullifier_exists_oracle(nullifier: Field) -> u8 {}
```

Comment:
Not sure I like renaming this `_oracle` 🤔 It is a Noir oracle, but only for the purpose of transpiling it to an AVM opcode. IMO `_opcode` would be a better name if we need a suffix.

Is the suffix needed because otherwise you'd have the same function name used twice in this namespace?

Reply:
Are you trying to use a name that makes sense for both the old public context and the avm context?

Reply:
I'll try to change it back. I changed it to avoid a recursive call in nullifier_exists (context). I think I had that problem in the test contract once but here it shouldn't be a problem.